ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1476,7 +1476,12 @@ vlan_interfaces:
         dr_priority: < 0-429467295 >
         sparse_mode: < true | false >
         local_interface: < local_interface_name >
+    # The below "VARPv6" keys will be deprecated in AVD v4.0 - These should not be mixed with the new "VARPv6" key above to avoid conflicts.
```

Comment:
```suggestion
    # The below "ipv6_virtual_router_address" key will be deprecated in AVD v4.0 - These should not be mixed with the new "ipv6_virtual_router_addresses" key below to avoid conflicts.
```